app.rb

```diff
@@ -0,0 +1,108 @@
+require_relative 'leshoyasoap'
+require 'colorize'
+# Ask for a username
+
+
+a =  Dir.glob("/Users/leshoyastarkey/Dropbox/SoapBox/*")
+
+a. each do |element|
```

Comment:
Typically the `a. each` would not have a space between the `.` and `each`. Although it's not required.